script.js

```diff
@@ -0,0 +1,131 @@
+document.getElementById('reloadAll_dn').addEventListener('click', reloadDone);
+
+reloadSum();
+
+//텍스트를 받아 삭제 버튼을 포함한 테이블의 한 행을 만들어주는 함수
+function addList(todoStr){
+    var tr=document.createElement('tr');
+    var td_txt=document.createElement('td');
+    td_txt.setAttribute('class', 'td_txt');
+    td_txt.innerHTML = todoStr;
+    tr.appendChild(td_txt);
+
+    var td_dlt=document.createElement('td');
+    td_dlt.setAttribute('class', 'td_dlt');
+    var input_dlt=document.createElement('input');
+    input_dlt.setAttribute('type', 'button');
+    input_dlt.setAttribute('value', "X");
+    input_dlt.setAttribute('class', 'btn_dlt');
+    input_dlt.onclick=dltList;
+    td_dlt.appendChild(input_dlt);
+    tr.appendChild(td_dlt);
+    return tr;
+}
+
+//입력창에서 텍스트만 받아내어 todoList()로 넘겨주는 함수
+function receiveText(){
+    var todoText = document.querySelector("#todoText");
+    if(!todoText.value){
+        alert('입력된 내용이 없습니다.');
+        todoText.focus();
+        return false;
+    }
+    var todoStr= todoText.value;
+    todoList(todoStr);
+}
+
+//입력된 텍스트를 todoList에 테이블 형식으로 추가시켜주는 함수
+function todoList(todoStr){
+    var tr = addList(todoStr);
+
+    var td_chk=document.createElement('td');
+    td_chk.setAttribute('class', 'td_chk');
+    var input_chk=document.createElement('input');
+    input_chk.setAttribute('type', 'button');
+    input_chk.setAttribute('value', "완료");
+    input_chk.setAttribute('class', 'btn_chk');
+    input_chk.onclick=doneList;
+    td_chk.appendChild(input_chk);
+    tr.appendChild(td_chk);
+
+    document.getElementById("todoTableBody").appendChild(tr);
+    todoText.value='';
+    todoText.focus();
+
+    reloadSum();
+}
+
+// 'X' 버튼을 누르면 해당 테이블 열을 삭제시켜주는 함수
+function dltList(){
+    this.parentElement.parentElement.remove();
+    reloadSum();
+}
+
+// todoList에서 완료 버튼을 누르면 doneList로 테이블을 옮겨주는 함수
+function doneList(){
+    this.parentElement.parentElement.remove();
+    var tdText=this.parentElement.parentElement.innerText;
+    var tr = addList(tdText);
+
+    var td_rtn=document.createElement('td');
+    td_rtn.setAttribute('class', 'td_chk');
+    var input_rtn=document.createElement('input');
+    input_rtn.setAttribute('type', 'button');
+    input_rtn.setAttribute('value', "복귀");
+    input_rtn.setAttribute('class', 'btn_rtn');
+    input_rtn.onclick=rtrnTodo;
+    td_rtn.appendChild(input_rtn);
+    tr.appendChild(td_rtn);
+
+    document.getElementById("doneTableBody").appendChild(tr);
+    reloadSum();
+}
+
+//doneList에서 복귀 버튼을 누르면 다시 todoList로 복귀시켜주는 함수
+function rtrnTodo(){
+    this.parentElement.parentElement.remove();
+    todoList(this.parentElement.parentElement.innerText);
+    reloadSum();
+}
+
+//todo와 done 테이블 개수를 업데이트시켜 보여주는 함수
+function reloadSum(){
+    var todoTable=document.getElementById("todoTableBody");
+    var todo_sum = todoTable.childElementCount;
+    var todo_div = document.createElement("div");
+    todo_div.innerHTML = "&ensp;To-Do (" + todo_sum + ")";
```

Comment:
문자열을 이어붙일 때 `+` 연산자 사용 대신, 템플릿 리터럴을 사용하면 더욱 깔끔한 코드를 작성하실 수 있어용
```suggestion
    todo_div.innerHTML =  `&ensp;To-Do (${todo_sum})`;
```